src/pages/ReimbursementAccount/NonUSD/SignerInfo/index.tsx

```diff
@@ -101,14 +142,17 @@ function SignerInfo({onBackButtonPress, onSubmit}: SignerInfoProps) {
             setCurrentSubStep(SUBSTEP.IS_DIRECTOR);
         } else if (currentSubStep === SUBSTEP.HANG_TIGHT) {
             Navigation.goBack();
+        } else if (currentSubStep === SUBSTEP.ARE_YOU_DIRECTOR) {
+            setCurrentSubStep(SUBSTEP.SIGNER_DETAILS_FORM);
         } else {
             setCurrentSubStep((subStep) => subStep - 1);
         }
-    };
+    }, [currentSubStep, goToTheLastStep, isEditing, isUserDirector, onBackButtonPress, prevScreen, screenIndex]);
 
-    const handleEmailSubmit = () => {
+    const handleEmailSubmit = useCallback(() => {
+        // TODO: the message to the email provided in the previous step should be sent
```

Comment:
Not sure what this is? Isn't think done in the `AskForCorpaySignerInformation` command?

Reply:
yes - this is placeholder for handling this endpoint, we do it in other PR, as this was intented to be done as separate issue

Reply:
sounds good 👍🏽